mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

```diff
@@ -100,6 +127,22 @@ def select(self, query: ast.Select) -> pd.DataFrame:
             where_conditions,
             order_by_conditions
         )
+        orders_df = select_statement_executor.execute_query()
```

Comment:
This code (130-145 lines) should be under `class OrdersTable(APITable):` not here.

Please correct it and make sure it works.

Reply:
There must have been some mistakes when merging the staging branch. Please address those.

Reply:
updated